regression.py

```diff
@@ -705,6 +706,9 @@ class GAMESSUSIRTest_ts(GenericIRTest):
     @unittest.skip('This is a transition state with different intensities')
     def testirintens(self):
         """This is a transition state with different intensities."""
+    @unittest.skip('This is a transition state with different atomcoords')

```

Comment:
I found a bug in the GAMESSUK parser - the parsed atomcoords from the file data/GAMESS-UK/basicGAMESS-UK8.0/dvb_ir.out test were in Bohr instead of Angstrom, while the atomcoords in data/GAMESS-UK/basicGAMESS-UK8.0/dvb_ir.outdvb_gopt_ks.out  were being parsed correctly. Seemingly sometimes GAMESS-UK prints coordinates in Bohr, sometimes in Angstrom, and cclib was not detecting which units were used and doing the necessary conversion. I fixed that in recent cclib pull request (which had not yet made it onto the cclib master, as Karol asked me to split the pull request into several smaller ones, and I'm making one pull request at a time). As this bug went undetected  - only GenericGeoOptTest was testing coordinates, I decided to test atom coords within the GenericIRTest, so that any similar bugs for other codes would not slip through the tests. As the atomcoord test is specific to the ground state dvb optimized geometry, it can fail for other geometries, e.g. a transition state - hence, why that test needs to be skipped for the file GAMESS/GAMESS-US2006/nh3_ts_ir.2006.2.22.r2.out, which this test runs against.